pkgs/servers/roon-server/default.nix

```diff
@@ -0,0 +1,60 @@
+{ alsaLib, alsaUtils, cifs-utils, fetchurl, ffmpeg, libav, mono, stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "roon-server";
+  version = "100600401";
+
+  src = fetchurl {
+    url = "http://download.roonlabs.com/updates/stable/RoonServer_linuxx64_${version}.tar.bz2";
+    sha256 = "121mmdh35q4bpgsqhcps6a6q1f4ld9v4hq9gp181bf2n779pk8sh";
+  };
+
+  installPhase = ''
+    # Check script
+    sed -i '3i PATH=$PATH:${cifs-utils}/bin:${ffmpeg}/bin:${libav}/bin' check.sh
```

Comment:
You can do this instead: ```PATH=$PATH:${lib.makeBinPath [ cifs-utils ffmpeg libav ]}```

Reply:
And alternative (it really depends on what's easier to maintain), is to use ```wrapProgram``` on the script as sed statements can be a real pain when things change upstream.

Reply:
I'm going to continue using `sed` for now since I'm also using it to edit the contents of the script, but I may update in the future if maintenance becomes burdensome.

Updated to use `lib.makeBinPath`